SlicerDeCA.json

```
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Slicer/Slicer/main/Schemas/slicer-extension-catalog-entry-schema-v1.0.1.json#",
+  "build_dependencies": [],
+  "build_subdirectory": ".",
+  "category": "SlicerDeCA",
+  "scm_revision": "main",
+  "scm_url": "https://github.com/SlicerMorph/SlicerDeCA.git",
+  "tier": 5
```

Since the extension does not currently have any automated tests for the DeCA module, it doesn't currently meet the following criteria necessary for Tier 5 status.

> - [ ] Automated tests for all critical features.


```suggestion
  "tier": 3
```